api/src/main/java/edu/cornell/mannlib/vitro/webapp/config/ConfigurationPropertiesSmokeTests.java

```diff
@@ -157,8 +159,9 @@ private void checkLanguages(ServletContext ctx, ConfigurationProperties props, S
 		if (selectableLanguages) {
 			List<String> selectableLanguagesList = Arrays.asList(selectString.split("\\s*,\\s*"));
 			for (String language : selectableLanguagesList) {
-				String vivoBundle = VIVO_BUNDLE_PREFIX + language + ".properties";
-				String vitroBundle = VITRO_BUNDLE_PREFIX + language + ".properties";
+				Locale locale = LocaleUtility.languageStringToLocale(language);
+				String vivoBundle = VIVO_BUNDLE_PREFIX + locale.stripExtensions().toString() + ".properties";
```

Comment:
.stripExtensions() is called in four places in this PR.  Not only does this seem brittle, but it's not intuitive to me why it's necessary.  If stripping extensions is something that needs to be done regularly, can this be put into a utility method with a comment explaining its purpose?